modules/vf-graphql-holochain/mutations/plan.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * plan mutations
+ *
+ * @package: HoloREA
+ * @since:   2019-05-27
```

Comment:
```suggestion
 * @package: hREA
 * @since:   2022-05-23
```

Reply:
of course we will be doing a full repo-wide sweep for HoloREA references anyways